.github/workflows/Tutor_CI.yml

```diff
@@ -0,0 +1,32 @@
+name: Tutor_CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.7]
```

Comment:
I don't think you need this matrix stuff here - you don't have plans to support more then one Python version do you?